src/algorithms/highlevel/sbic.cpp

```diff
@@ -253,31 +253,24 @@ void SBic::compute()
         bicChangeResult = bicChangeSearch(kernel, _inc1, startKernel, dmin);
         std::vector<Real> tmpBicValues(bicChangeResult.second);
         
-        if (i = bicChangeResult.first)
+        // Store bic values up to (and including) the peak
```

Comment:
Change the comment to:
Store bic values over the whole kernel (no min was detected) or up to the index of the min